src/picongpu/include/fields/background/templates/TWTS/BField.hpp

```diff
@@ -40,7 +40,21 @@ class BField
 {
 public:
     typedef float_X float_T;
-    
+
+    enum PolarizationType
+    {
+        /* The linear polarization of the TWTS laser is defined
+         * relative to the plane of the pulse front tilt (reference plane). */
```

Comment:
Is there a reason why you write very often two separate comment blocks instead of one?
I marked the positions with a short comment, I am not sure if this should changed or not. 

IMO the comment over a code part corresponds to the following part: e.g.
```C++
/** set variable to zero*/
int x=0;
```

Example with two comment blocks:
```C++
/** this block has no code reference*/
/** set variable to zero*/
int x=0;
```